app/assets/javascripts/js-routes.js.erb

```diff
@@ -1,3 +1,3 @@
 <%# encoding: UTF-8 %>
-<% depend_on "file://#{Rails.root.join 'config', 'routes.rb'}" if JsRoutes::SPROCKETS3 %>
+<% depend_on Rails.root.join('config', 'routes.rb') if JsRoutes::SPROCKETS3 %>
```

Comment:
My proposed PR fixes the problem. No need to convert 'Pathname'

On Mon, 25 Jan 2016 at 22:25 Alexey Vasiliev <notifications@github.com>
wrote:

> In app/assets/javascripts/js-routes.js.erb
> <https://github.com/railsware/js-routes/pull/178#discussion_r50691543>:
>
> > @@ -1,3 +1,3 @@
> >  <%# encoding: UTF-8 %>
> > -<% depend_on "file://#{Rails.root.join 'config', 'routes.rb'}" if JsRoutes::SPROCKETS3 %>
> > +<% depend_on Rails.root.join('config', 'routes.rb') if JsRoutes::SPROCKETS3 %>
>
> I think you need convert Pathname object to string by to_s method:
>
> Rails.root.join('config', 'routes.rb').to_s
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/railsware/js-routes/pull/178/files#r50691543>.
>